rb/poll.py

```diff
@@ -157,6 +157,15 @@ def poll(self, timeout=None):
         return rv
 
 
+def is_closed(f):
+    poller = poll()
```

Comment:
Is it bad that we're creating a new poller instance for every connection check just to throw it away?